packages/ui/packages/Grid/test/index.spec.js

```diff
@@ -29,4 +29,16 @@ describe('Grid ui component tests', () => {
     const gridChild = getByTestId('gridChild');
     expect(getByTestId('grid')).toContainElement(gridChild);
   });
+
+  it('Should render responsive children', () => {
+    const { getByTestId } = render(
+      <Wrapper>
+        <Grid perRow={{xs: 12, md: 6}}>
```

Comment:
Is there a way to test rendering `xs: 12, md: 6` too?

Reply:
it seems you did it in another PR here: https://github.com/snappmarket/frontend-toolbox/pull/220

Reply:
to test the sizes in different viewports we shall simulate the window resize, but there's no need to do that cause the grid item sizes are already covered by the first test case in this file
`Should render grid with perRow of 4`

about the second comment, that was for the grid system component and tested the container, row, and col components, and this one is for the grid component.